bcgov/HOUS-permit-portal
# Add submission inbox setup screen

[BPHH-788]

## Description

Adds screen for configuring submission inbox emails per permit type. Emails must be verified to be used as the submission email (currently there is no guard but this will be added in [BPHH-841](https://hous-bssb.atlassian.net/browse/BPHH-841))

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Implements [BPHH-788](https://hous-bssb.atlassian.net/browse/BPHH-788).

## Steps to QA

1. Login as review manager
2. Navigate to `Configuration Management` ➡️ `Submission Inbox Setup`

Verify:
As a REVIEW MANAGER:
1. user can add and update submission email for each permit type
2. user can add multiple submission emails for a permit type 
3. user can remove submission emails
4. confirmation email is sent to submission emails on create OR update
5. UI status is updated once email is confirmed so alert no longer shows up

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

N/A
